test/mmap.jl

```diff
@@ -278,3 +278,12 @@ n = similar(m, 12)
 @test length(n) == 12
 @test size(n) == (12,)
 finalize(m); m = nothing; gc()
+
+file = tempname()
+touch(file)
+open(file, "r+") do s
+    A = Mmap.mmap(s, Vector{UInt8}, (10,), 0);
+    Mmap.sync!(A)
+    A = Mmap.mmap(s, Vector{UInt8}, (10,), 1);
+    Mmap.sync!(A)
+end
```

Comment:
eperm, might need to force a gc? https://ci.appveyor.com/project/StefanKarpinski/julia/build/1.0.13049/job/mx2upgjkcduxik5v